This patch is required to make procmail cross compile - it tries to run some automatic tests which
obviously do not work in a cross compiled environment.


--- /dev/null
+++ b/autoconf.h
@@ -0,0 +1,21 @@
+/* This file was automagically generated by autoconf */
+
+/* 5 moves in 64 steps of size 16384 when reallocing */
+#define NOpw_class
+#define NOstrlcat
+#define NOsetrgid
+#define MAX_argc 7588
+/* Your system's strstr() is 1.12 times FASTER than my C-routine */
+#define UDP_protocolno 17
+#define BIFF_serviceport "512"
+#define IP_localhost {127,0,0,1}
+#define MAILSPOOLDIR "/var/spool/mail/"
+#define SENDMAIL "/usr/sbin/sendmail"
+#define CF_no_procmail_yet
+#define buggy_SENDMAIL
+#define defPATH "PATH=$HOME/bin:/bin:/usr/bin:/usr/local/bin:/usr/bin/X11"
+#define defSPATH "PATH=/bin:/usr/bin:/usr/local/bin:/usr/bin/X11"
+#define PM_VERSION "3.22"
+/* Hotwire LOCKINGTEST=100 */
+/* Procmail will lock via: dotlocking, fcntl() */
+/* autoconf completed */
--- /dev/null
+++ b/Makefile.openwrt
@@ -0,0 +1,18 @@
+PM_OBJ=cstdio.o common.o exopen.o goodies.o locking.o \
+ mailfold.o foldinfo.o misc.o pipes.o regexp.o robust.o \
+ sublib.o acommon.o mcommon.o lastdirsep.o authenticate.o \
+ lmtp.o memblk.o variables.o from.o comsat.o
+
+FM_OBJ=common.o fields.o formisc.o sublib.o ecommon.o \
+ acommon.o
+
+LDFLAGS += -lm -ldl -lc
+CFLAGS += -Os -DPROCMAIL
+
+all: procmail formail
+
+procmail: procmail.o $(PM_OBJ)
+	$(CC) $(CFLAGS) $@.o $(PM_OBJ) -o $@ $(LDFLAGS)
+
+formail: formail.o $(FM_OBJ)
+	$(CC) $(CFLAGS) $@.o $(FM_OBJ) -o $@ $(LDFLAGS)
